### PR TITLE
refactor(schemas, console): alter the resource scopes description field to nullable

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Alter the resource scopes description field to nullable. Align the resource scopes structure with organization scopes. 

related PR #5503 

## Testing

test locally

## Checklist

- [x] `.changeset`
- [ ] ~unit tests~
- [ ] ~integration tests~
- [ ] ~necessary TSDoc comments~

### CODE REVIEW

File: packages/console/src/pages/ApiResourceDetails/ApiResourcePermissions/components/CreatePermissionModal/index.tsx [hunk 1]
```diff
@@ -118,10 +118,9 @@ function CreatePermissionModal({ resourceId, totalResourceCount, onClose }: Prop
               error={errors.name?.message}
             />
           </FormField>
-          <FormField isRequired title="api_resource_details.permission.description">
+          <FormField title="api_resource_details.permission.description">
             <TextInput
               placeholder={t('api_resource_details.permission.description_placeholder')}
-              {...register('description', { required: true })}
```
Comment (charIeszhao) [inline packages/console/src/pages/ApiResourceDetails/ApiResourcePermissions/components/CreatePermissionModal/index.tsx:124]: Why removing `...register('description')`?

Comment (simeng-li) [reply packages/console/src/pages/ApiResourceDetails/ApiResourcePermissions/components/CreatePermissionModal/index.tsx:124]: looks like a miss deletion. should remove the required only.

Comment (simeng-li) [reply packages/console/src/pages/ApiResourceDetails/ApiResourcePermissions/components/CreatePermissionModal/index.tsx:124]: updated

File: .changeset/seven-socks-perform.md [hunk 2]
```diff
@@ -2,23 +2,24 @@
 "@logto/schemas": patch
 ---
 
-## Fix third-party app get /interaction/consent 500 bug
+## Resolve Third-Party App's /interaction/consent Endpoint 500 Error
```
Comment (gao-sun) [inline .changeset/seven-socks-perform.md:5]: does it have a rebase issue?

Comment (simeng-li) [reply .changeset/seven-socks-perform.md]: By design. This PR will overwrite the previous changeset update.   It's a better fix, which will align the type definition.

**Pull request merged**